### DISCUSSION TITLE
Adding lightning sensor to dashboard

### DISCUSSION THREAD

**Discussion by cbirkenbeul** [Daveiano/weewx-wdc#252] [Q&A] [answered]
Hello,
I bought a lightning detector for my ecowitt weather station. The sensor is detected and weewx seems to be reading the data. I got pushed values to mqtt.

What should I do I show the data from this sensor in the wdc website? I leave the skins.conf default. As I guessed, it should be display automatically, but it does not. I already have deleted the html files, just to be sure that is no caching problem. 
Thanks.

**Comment by Daveiano** [comment, marked as answer]
Hello there,

> What should I do I show the data from this sensor in the wdc website? I leave the skins.conf default. As I guessed, it should be display automatically, but it does not.

I assume you want to show `lightning_strike_count`? If so you need to add it to either `stat_tile_observations`, `table_tile_observations` or `diagrams`. The observation is not included by default.

Did the sensor already capture lightning strikes? Every Display of an observation (as chart or inside of a stat tile) is enclosed by an check if the observation has data. I don't know how `lightning_strike_count` is handled (could be `null` if nothing was captured) but it could be the case that it is not shown unless the sensor captured some data.

**Reply by cbirkenbeul** [reply]
Thanks for you reply.
After adding the lightning_strike_count to the fields I now see them on the website. Did you know if it's also possible to show the distance? Or is that not possible with your skin?

**Reply by Daveiano** [reply]
Then you will need to add the observation `lightning_distance`.
